R/get_inboveg_header.R

```diff
@@ -109,10 +109,10 @@ get_inboveg_header <- function(connection,
   , ivR.VagueDateEnd
   , ivR.SurveyId
   , ivR.RecTypeID
+  , ivR.NeedsWork
   FROM [dbo].[ivRecording] ivR
   INNER JOIN [dbo].[ivSurvey] ivS on ivS.Id = ivR.SurveyId
-  INNER JOIN [dbo].[ivRecTypeD] ivRec on ivRec.ID = ivR.RecTypeID
-  where ivR.NeedsWork = 0"
+  INNER JOIN [dbo].[ivRecTypeD] ivRec on ivRec.ID = ivR.RecTypeID"
 
```

Comment:
removing the `where` statement will give trouble because other `where` statements are pasted to the `common_part` of the sql statement. Maybe try changing line 115 to `where 1=1`

Reply:
Another option would be to replace the `AND` at the beginning of the 2 pasted parts by `WHERE`.  Advantage is that we wouldn't ask ourselves in the future why this `1=1` is added (and it is less verbose).

Reply:
is there a tric to control the functions you adapted? now I always copy them in a seperate r-file and try to run, but often this need a lot of adjustments (it works of course, but more efficient would be when testing is possible in this project itself).

Reply:
@ElsLommelen "where 1 = 1 " is often used in sql-scripts (allez dat is wat Jo Loos me eigenlijk zo geleerd heeft, dus dom dat ik daar hier niet zelf aan gedacht had).

Reply:
Normally you have in this project in your right upper corner a tab 'build'.  Clicking the button 'install and restart' installs the adapted version of the package on your computer.  And then you can for instance run the example by putting your cursor on it and hit crtl-enter.

Reply:
Ah, ok, wist ik niet, en de reden is gewoon om die volgende regel niet te moeten aanpassen?